### DISCUSSION TITLE
Is next-auth production ready? Would you use it for making a new Saas?

### DISCUSSION THREAD

**Discussion by znwhite** [nextauthjs/next-auth#2736] [Help]
Just a general discussion question. I was wondering if people think next-auth is production ready, ie if you were making a new SaaS product would you be confident using next-auth to handle your app sign/signout etc? Looking forward to hearing your opinions.

**Comment by balazsorban44** [comment]
I would say it is production-ready as it is one of the most popular libraries in its category, with ~300.000 downloads a month. 😅

If it fits your specific use case is a totally different question and would need more details to be able to decide/recommend.

If you are talking about the upcoming v4 release, it is in public beta, and we intend to promote it to stable if some of the last checks will pass, hopefully in less than a month.
